Automattic/wp-calypso
# E2E: fix permafailing step in Block Widgets spec.

#### Changes proposed in this Pull Request

This PR corrects the permanently failing step in the Block Widgets spec for WPCOM Gutenberg build.

Key changes:
- create new component class, `BlockWidgetEditorComponent`.
- implement method to dismiss any modals or tours that appear when the BlockWidgetEditorComponent is loaded.

#### Testing instructions

Check the WPCOM Gutenberg Playwright builds (mobile, desktop) and ensure that:
- [x] Block Widget test is not failing;

Related to https://github.com/Automattic/wp-calypso/issues/59930
